plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/project/DataflowProjectCreator.java

```diff
@@ -194,31 +195,34 @@ public void run(IProgressMonitor monitor)
             mavenGroupId, mavenArtifactId, "0.0.1-SNAPSHOT", packageString, archetypeProperties,
             projectImportConfiguration, progress.newChild(4));
         break;
-      } catch (CoreException e) {
-        failures.add(e);
+      } catch (CoreException ex) {
+        // todo should we use a MultiStatus here?
```

Comment:
It would seem so!  Given we're potentially trying multiple archetype versions, it maybe should be a nested set of MultiStatus objects.